src/store/util/fetchUtils.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+/**
+ * A module for middleware that would like to make external calls through `fetch`
+ */
+export const parseQueryResponse = (queries: Array<Query>) => (
```

Comment:
Is it worth writing more tests for this, or do we feel like we've got pretty good coverage with the fetchQueries tests?